docs/toolbox/accessibility_indicators/heatmaps/gravity.md

```diff
@@ -9,33 +9,34 @@ import thematicIcon from "/img/toolbox/data_management/join/toolbox.webp";
 
 ## 1. Explanation
 
-Visualized as a color-coded hexagonal grid, heatmaps take into account real-world transport and street networks to compute accessibility. When you specify a routing mode (such as `walking`, `bicycle`, etc.) and a maximum travel duration, a grid reflecting all areas accessible under these conditions is produced.
+Visualized as a color-coded hexagonal grid, heatmaps takes into account real-world transport and streets networks, to compute accessibility. Specify a routing mode (`Walk` or `Bicycle`), select `Opportunity Layer`. Set a maximum travel duration. The result shall display the color-coded hexagonal grid for all the areas accessible under those conditions. The color scale referes to the local accessibility.
+
+:::info INFO
+The `Opportunity Layer` is a layer with points data type. View more in [**Data Types**](../data/data_types).
+:::
 
 Unique to the **Gravity-based Heatmap**, customizable properties such as *sensitivity*, the *impedance function* and *destination potential* give you minute control over the method used and metadata taken into account while computing the accessibility value for an area. Influenced by these properties, the accessibility of a point can model complex real-world human behaviour and is a powerful measure for transport and accessibility planning.
 
 :::tip Pro tip
-
-Described succinctly, heatmaps are a visualization representing *access* from various unspecified origins to one or more specified destinations. This is in contrast to catchment areas which represent *egress* from one or more specified origins to various unspecified destinations.
-
+Described succinctly, Accessibility Heatmaps are a visualization representing *access* from various unspecified origins, to one or more specified destinations. This is in contrast to catchment areas which represent *egress* from one or more specified origins to various unspecified destinations.
 :::
 
-TODO: Update this image
-
-![Catchment Area in GOAT](/img/toolbox/accessibility_indicators/catchments/catchment_sample.png "Gravity-based Heatmap in GOAT")
+![Gravity-based Heatmap in GOAT](/img/toolbox/accessibility_indicators/heatmaps/gravity_based/heatmap_gravity_based.webp "Gravity-based Heatmap in GOAT")
   
 :::info 
-The calculation of heatmaps is currently possible in specific regions. Upon selecting a routing mode, you will see a **geofence** on the map indicating supported regions.
+The Heatmaps are available for specific regions. 
+Upon selecting a `Routing Mode` the **geofence** for heatmap will be displayed on the map, indicating supported regions.
 
-TODO: Update this image to reflect the latest geofence for heatmaps
 <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
   <img src={require('/img/toolbox/accessibility_indicators/catchments/geofence.png').default} alt="Geofence for catchment area calculation in GOAT" style={{ maxHeight: "400px", maxWidth: "400px", alignItems:'center'}}/>
```

Comment:
I think this still needs to be updated?

Reply:
Heatmaps currently support only Germany. You can maybe use the same image of the public transport catchment area geofence.

Reply:
For some reason it doesn't show a Germany only geofence for me. If you could provide de image.